### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by fuhrmanator**

### CODE REVIEW

File: userdocumentation.html [hunk 1]
```diff
@@ -84,9 +84,9 @@ <h1 class="doc-title"><i class="icon fa fa-user"></i> User Documentation</h1>
                             <h2 class="section-title">Introduction</h2>
                             <div class="section-block">
                                 <p>
-                                    Moose-Query is a domain-specific language to build queries for entities in <a
+                                    Moose-Query is an internal domain-specific language (DSL) to build queries for entities in <a
                                         href="http://www.moosetechnology.org/" class="text-highlight">Moose</a>. It
-                                    succeeds to Moose-Chef. It is designed to simplify and standardize the way to query Moose
+                                    replaces Moose-Chef, and is designed to simplify and standardize the way to query Moose
```
Comment (jecisc) [inline userdocumentation.html:89]:
```suggestion
                                     replaces Moose-Chef, and is designed bla to simplify and standardize the way to query Moose
```

**Comment by jecisc** [comment]
@fuhrmanator Hi,

Is this PR ready? I think you opened it so that we can check some things. 

Also, I'm not a big fan of the full reformating on the api file because Nicolas opened another PR and it will conflict with the reformating probably.

**Comment by fuhrmanator** [comment]
OK, @jecisc, I reverted the one with formatting. I was using a WYSIWYG HTML tool to make it easier to change the content, and so it changed the HTML too much. Sorry about that.

**Comment by jecisc** [comment]
Thank you!! :)

**Pull request merged**